sdks/python/build.gradle

```diff
@@ -272,61 +308,67 @@ task portableWordCount(dependsOn: 'installGcpTest') {
   }
 }
 
+// Run single or a set of integration tests with provided test options and pipeline options.
 task integrationTest(dependsOn: ['installGcpTest', 'sdist']) {
   doLast {
-    def cmd_args = []
+    def argMap = [:]
 
     // Build test options that configures test environment and framework
-    def test_options = [
-            "--nocapture",
-            "--nologcapture",
-            "--processes=8",
-            "--process-timeout=3000",
-    ]
+    def testOptions = basicTestOpts
     if (project.hasProperty('testOptions'))
       // Customize test options by adding more nose flags to -Ptest_opt in commandline
-      test_options = project.property('test_options')
+      testOptions = project.property('testOptions')
     if (project.hasProperty('attr'))
-      test_options += ["--attr=${project.property('attr')}"]
+      testOptions += ["--attr=${project.property('attr')}"]
     if (project.hasProperty('tests'))
-      test_options += ["--tests=${project.property('tests')}"]
-    cmd_args += ["--test_opts \"${test_options.join(' ')}\""]
+      testOptions += ["--tests=${project.property('tests')}"]
+    argMap["test_opts"] = testOptions
 
     // Build pipeline options that configures pipeline job
     if (project.hasProperty('pipelineOptions'))
-      cmd_args += ["--pipeline_opts \"${project.property('pipelineOptions')}\""]
+      argMap["pipeline_opts"] = project.property('pipelineOptions')
 
+    def cmdArgs = mapToArgString(argMap)
     exec {
       executable 'sh'
-      args '-c', ". ${envdir}/bin/activate && ./scripts/run_integration_test.sh ${cmd_args.join(' ')}"
+      args '-c', ". ${envdir}/bin/activate && ./scripts/run_integration_test.sh $cmdArgs"
     }
   }
 }
 
-task postCommitITTests(dependsOn: 'installGcpTest') {
+// Run PostCommit integration tests on default runner (TestDataflowRunner)
+task postCommitIT(dependsOn: ['installGcpTest', 'sdist']) {
```

Comment:
Is it possible to exclude some test cases from the postCommitIT?

Reply:
I would suggest to use `integrationTest` in [line 312](https://github.com/apache/beam/pull/6833/files/7632ca4e412696747730df801d55f9a3e58e6750#diff-5924c115647bc4008928b5855e17d635R312) if you have a specific set of tests to run. You can use `-Ptests` or `-Pattr` in gradle command.

Reply:
integrationTest doesn't seem to be used. Is it for manual invocations? Are there any docs on how to use it?

Reply:
`PerformanceTest_Python` used it to invoke single test/pipeline (wordcount_it_test). I don't know if any contributor guide references it, but guess it could be a good example to run single integration test (like wordcount_it_test) in new contributor guide.